### PR TITLE
Ported TubeFilter and ColoredElevationMap from Cxx to Python

### PR DESCRIPTION
**Pull request by pawelkurzynski**
Hi,
I ported two examples from Cxx to Python. This is my first contribution to this project. I have just started working with VTK.

### CODE REVIEW

File: src/Python/Meshes/ColoredElevationMap.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+def main():
+
+    # Create a grid points
+    points = vtk.vtkPoints()
+    GridSize = 20;
+    xx=0.0 
+    yy=0.0
+    zz=0.0
+    for x in range(0, GridSize):
```
Comment (ajpmaclean) [inline src/Python/Meshes/ColoredElevationMap.py]: Please use `vtkMinimalStandardRandomSequence` here in place of `vtkMath.Random`. See: [OrientedCylinder](https://lorensen.github.io/VTKExamples/site/Python/GeometricObjects/OrientedCylinder/) for an example using `vtkMinimalStandardRandomSequence`.

**Comment by ajpmaclean** [comment]
Looking good, some more comments/fixes:

- You may need to experiment with the random seed.
- Additionally please supply a png image for each example. For your convenience a snippet [WriteImage](https://lorensen.github.io/VTKExamples/site/Python/Snippets/WriteImage/) is provided to help you write out a test image. Be sure to remove this snippet once the image is written out.
- Test images must follow this naming convention: **Test<Example_Name>.png**, for example: **TestColoredElevationMap**.
- For these two examples, the test images will go in `src/Testing/Baseline/Python/Meshes` and `src/Testing/Baseline/Python/PolyData` respectively.
- If there is no test image, or it is not correctly named, or it is not in the correct folder; there will be no image with your example when the web pages are generated from the source.

File: src/Python/Meshes/ColoredElevationMap.py (hunk 1 above)
Comment (pawelkurzynski) [reply src/Python/Meshes/ColoredElevationMap.py]: Thank you. I added this request.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**